docs/dev-machine-setup.md

```diff
@@ -0,0 +1,109 @@
+     ```
+     * Alternatively, if the project you want to use already exists, just switch to the new project using:
+        ```
+        $ gcloud config set project <your-project>
+        ```
+  1. The next step is to create a VM image able to run nested KVM (as outlined [here](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances)). **IMPORTANT:** Notice that Firecracker requires a relatively new kernel, so you should use a recent Ubuntu 18 image (or equivalent).
+     ```
+     $ gcloud compute disks create disk-ub18 --image-project ubuntu-os-cloud --image-family ubuntu-1804-lts
+     ```
+     If you are prompted whether to enable the Compute API for this project, confirm, wait until prompted again, select region - e.g., us-east1-b
+     ```
+     $ gcloud compute images create ub18-nested-kvm --source-disk disk-ub18 --licenses "https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx" --source-disk-zone us-east1-b
+     ```
+     Make sure you run it as a single command, and that you specify the right region.
+  1. Now we create the VM:
+     ```
+     $ gcloud compute instances create firecracker-vm --zone us-east1-b --image ub18-nested-kvm
+     ```
+  1. Connect to the VM via SSH.  
+     ```
+     $ gcloud compute ssh firecracker-vm
+     ```
+     When doing it for the first time, a key-pair will be created for you (you will be propmpted for a passphrase - can just keep it empty) and uploaded to GCE. Done! You should see the prompt of the new VM: `ubuntu@firecracker-vm:~$`  
+  1. Verify that VMX is enabled, install and enable KVM
+     ```
+     $ grep -cw vmx /proc/cpuinfo
+     1
+     $ sudo apt-get update && sudo apt-get install qemu-kvm -y
+     $ sudo setfacl -m u:${USER}:rw /dev/kvm
+     $ [ -r /dev/kvm ] && [ -w /dev/kvm ] && echo "OK" || echo "FAIL"
+     OK
+     ```   
+  1. Download Firecracker (based on instructions [here](https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md)). Optionally change the first URL below to the release you want, from https://github.com/firecracker-microvm/firecracker/releases
+     ```
+     $ curl -fsSL -o ./firecracker https://github.com/firecracker-microvm/firecracker/releases/download/v0.11.0/firecracker-v0.11.0
+     $ chmod +x ./firecracker
+     $ curl -fsSL -o ./hello-vmlinux.bin https://s3.amazonaws.com/spec.ccfc.min/img/hello/kernel/hello-vmlinux.bin
+     $ curl -fsSL -o ./hello-rootfs.ext4 https://s3.amazonaws.com/spec.ccfc.min/img/hello/fsfiles/hello-rootfs.ext4
+     ```
+  1. Run Firecracker
+     1. Run the VMM (if everything is working, it will NOT return to your command prompt):
+        ```
+        $ rm -f /tmp/firecracker.sock && ./firecracker --api-sock /tmp/firecracker.sock
+        ```
+     1. Open an additional terminal, SSH into your `firecracker-vm`. Configure and start the Guest VM (ignore the 'No Content' outputs):
```

Comment:
This is all covered by https://github.com/firecracker-microvm/firecracker/blob/master/docs/getting-started.md

Reply:
Yup, you can just use a relative link to the correct section within that document. Actually, once we have instructions for setting up dev hosts on various platforms, it doesn't even make sense to include this part here more. The natural conclusion for any of set up steps is to continue onto the `getting-started.md` doc (which in turn would link to this doc). So, we should just end this doc with a reference back to `getting-started.md`.